### PR TITLE
Support PRs with multiple changed files

### PR DESCRIPTION
**Pull request by dgurkaynak**
Highlighting was working for just first changed file in
`Files Changed` tab in pull requests.

### CODE REVIEW

File: src/highlight-selected.js [hunk 1]
```diff
@@ -28,10 +28,15 @@ window.addEventListener('load', function() {
...
             }
         }
-        var el = document.querySelector(query);
-        if (!el) return;
+        var elements = document.querySelectorAll(query);
+        if (elements.length == 0) return;
         textNodes = [];
-        getTextNodesIn(el);
+
+        for (var i = elements.length; i--;) {
```
Comment (xhacker) [inline src/highlight-selected.js]: Can you use a [`for..of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) statement here?

**Comment by xhacker** [comment]
Thank you for your pull request! Just a minor comment.

File: src/highlight-selected.js (hunk 1 above)
Comment (dgurkaynak) [inline src/highlight-selected.js]: Updated with for-of loop 👍

**Comment by xhacker** [comment]
:shipit:

**Pull request merged**